NEWS

```diff
@@ -40,6 +40,15 @@ New Functionality
 
 - The ``find_str`` and ``rfind_str`` bifs now support case-insensitive searches.
 
+- Added a new plugin hook for capturing packets that made it through analysis
+  without being processed called ``Plugin::HookUnprocessedPacket``. Currently
+  only ARP packets or packets with a valid IP-based transport header are marked
```

Comment:
Would remove the `only` because it makes it sounds like there's something substantial missing.